src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -51,20 +51,13 @@ The .NET Foundation licenses this file to you under the MIT license.
     <NativeObjectExt Condition="'$(TargetOS)' == 'windows'">.obj</NativeObjectExt>
     <NativeObjectExt Condition="'$(TargetOS)' != 'windows'">.o</NativeObjectExt>
 
-    <LibFileExt Condition="'$(TargetOS)' == 'windows'">.lib</LibFileExt>
```

Comment:
This file is copied into the NativeAOT package. @MichalStrehovsky have not brought the packaging infrastructure over yet, so it is not easy to see.

If it were to depend on other files in the repo, those other files need to be copied into the NativeAOT package as well that gets complex fast. I think it is better to leave it self-contained.

Reply:
Makes sense. I've reverted NativeAOT changes.